.github/workflows/scheduled.yaml

```diff
@@ -29,7 +30,50 @@ jobs:
 
       - name: Run Tests
         run: |
-            cargo nextest run \
-            --features test-clarity-v${{ matrix.clarity_version }} \
-            --test wasm-generation \
-            --test standard
+          set +e
+          cargo nextest run \
+          --features test-clarity-v${{ matrix.clarity_version }} \
+          --test wasm-generation \
+          --test standard 2>&1| tee output.log
+          test_exit=${PIPESTATUS[0]}
+          cat output.log | sed 's/\x1b\[[0-9;]*[a-zA-Z]//g' | sed 's/\x0//g' > cleaned_output.log
+          sed -n '/Test failed: assertion/,/FAIL/p' cleaned_output.log >> failure_section.log
+          echo "failure_section<<EOF" >> $GITHUB_ENV
+          cat failure_section.log >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+          if [ $test_exit -ne 0 ]; then
+            exit 101
+          fi
+
+      - name: Create GitHub Issue on Failure
+        uses: jayqi/failed-build-issue-action@v1
+        if: ${{ failure() }}
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          label-name: "property-testing-failure"
+          title-template: "Property testing failure"
+          body-template: |
+            GitHub Actions workflow [{{workflow}} #{{runNumber}}](https://github.com/{{repo.owner}}/{{repo.repo}}/actions/runs/{{runId}}) Clarity::V${{ matrix.clarity_version }} failed.
+            ```log
+            ${{ env.failure_section }}
+            ```
```

Comment:
@BowTiedWoo Can we use `${{ secrets.GITHUB_TOKEN }}` instead of `${{ secrets.GH_TOKEN }}?` `GITHUB_TOKEN` seems more secure and is automatically created for workflow use, with permissions configured directly in the workflow settings.

Can we simplify this by using the GitHub CLI command `gh issue create` instead of a third-party action for that purpose?

Reply:
I removed the third party action and it's now using `gh issue create` for creating the issue.